.dockerignore

```diff
@@ -0,0 +1,13 @@
+build/
+configure.ac
+COPYING*
+docs/
```

Comment:
docs/ is required now.  It may be good to install `man` (for alpine) or `man-db` (if switching to `debian:buster-slim` -- more on this later).

Reply:
For the builder that is okay, as this is all dropped on the second image, as we only install the binary.

Debian, ouch :) actually we can have and support both. It is very common to have 2 docker containers, one that uses debian (usually the default) targeted by :latest and one that targets alpine (small images) done by targeting :alpine.

I'll look into it if it is _really_ needed :) (below I suppose)